src/shared/mount-setup.h

```diff
@@ -10,3 +10,5 @@ int mount_cgroup_controllers(void);
 
 bool mount_point_is_api(const char *path);
 bool mount_point_ignore(const char *path);
+
+bool check_cg_feature_supported(void);
```

Comment:
still, please do not export this function, this has no use outside of mount-setup.c

make it a static function inside of mount-setup.c

Reply:
fixed.